.github/workflows/ci.yaml

```diff
@@ -13,7 +13,9 @@ jobs:
       - uses: actions/setup-go@v2
         with:
           go-version: '1.16'
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
What is this all about?

Reply:
It fetches tags, which we need for `git describe` (in `Makefile`).